Package.swift

```diff
@@ -17,10 +17,11 @@ let package = Package(
   name: "swift-testing",
 
   platforms: [
-    .macOS(.v13),
-    .iOS(.v16),
-    .watchOS(.v9),
-    .tvOS(.v16),
+    .macOS(.v10_15),
+    .iOS(.v13),
+    .watchOS(.v6),
+    .tvOS(.v13),
+    .macCatalyst(.v13),
```

Comment:
@Kyle-Ye, would you like to update these versions to match those I proposed in my [reply](https://github.com/apple/swift-testing/issues/101#issuecomment-1808673624) to #101?

Reply:
Sorry, I still do not get the point. Could you elaborate on why we need macOS 13 instead of macOS 10.15 and watchOS 7 instead of watchOS 6? See https://github.com/apple/swift-testing/issues/101#issuecomment-1809525632.

Reply:
Specifically, I maintain a package which needs to support the above platform version. The current swift-testing configuration will block me from migrating to use swift-testing.